<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="0" failed="1" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-02-13T19:49:10 IST" name="Default suite" finished-at="2023-02-13T19:49:18 IST" duration-ms="7527">
    <groups>
    </groups>
    <test started-at="2023-02-13T19:49:10 IST" name="Default test" finished-at="2023-02-13T19:49:18 IST" duration-ms="7527">
      <class name="com.marketplace.www.Selenium_Automation.SubmitOrderTest">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:com.marketplace.www.Selenium_Automation.SubmitOrderTest@1b7cc17c]" started-at="2023-02-13T19:49:10 IST" name="launchApplication" finished-at="2023-02-13T19:49:17 IST" duration-ms="6993" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="submitOrder()[pri:0, instance:com.marketplace.www.Selenium_Automation.SubmitOrderTest@1b7cc17c]" started-at="2023-02-13T19:49:17 IST" name="submitOrder" finished-at="2023-02-13T19:49:18 IST" duration-ms="461" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "org.openqa.selenium.WebElement.findElement(org.openqa.selenium.By)" because "prod" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.WebElement.findElement(org.openqa.selenium.By)" because "prod" is null
at rahulshettyacademy.pageobjects.ProductCatalogue.addProductToCart(ProductCatalogue.java:60)
at com.marketplace.www.Selenium_Automation.SubmitOrderTest.submitOrder(SubmitOrderTest.java:42)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrder -->
        <test-method signature="orderHistoryTest()[pri:0, instance:com.marketplace.www.Selenium_Automation.SubmitOrderTest@1b7cc17c]" started-at="2023-02-13T19:49:18 IST" name="orderHistoryTest" finished-at="2023-02-13T19:49:18 IST" depends-on-methods="com.marketplace.www.Selenium_Automation.SubmitOrderTest.submitOrder" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method com.marketplace.www.Selenium_Automation.SubmitOrderTest.orderHistoryTest() on instance com.marketplace.www.Selenium_Automation.SubmitOrderTest@1b7cc17c depends on not successfully finished methods 
[com.marketplace.www.Selenium_Automation.SubmitOrderTest.submitOrder() on instance com.marketplace.www.Selenium_Automation.SubmitOrderTest@1b7cc17c]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method com.marketplace.www.Selenium_Automation.SubmitOrderTest.orderHistoryTest() on instance com.marketplace.www.Selenium_Automation.SubmitOrderTest@1b7cc17c depends on not successfully finished methods 
[com.marketplace.www.Selenium_Automation.SubmitOrderTest.submitOrder() on instance com.marketplace.www.Selenium_Automation.SubmitOrderTest@1b7cc17c]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:150)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- orderHistoryTest -->
      </class> <!-- com.marketplace.www.Selenium_Automation.SubmitOrderTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
